### PR TITLE
adding Andreas Spanner to the member list

### PR DESCRIPTION
**Pull request by aspanner**
adding Andreas Spanner to the member list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +230,7 @@ Membership governance can be viewed [here](https://github.com/cncf/sig-security/
 * Abhishek Singh ([@araalinetworks](https://github.com/araalinetworks)), Araali Networks
 * Aeva Black ([@AevaOnline](https://github.com/AevaOnline)), Microsoft
 * Frederick Fernando ([@freddyfernando](https://github.com/freddyfernando)), InfraCloud
+* Andreas Spanner ([@aspanner] (https://github.com/aspanner)), Red Hat
```
Comment (TheFoxAtWork) [inline README.md:233]:
```suggestion
* Andreas Spanner ([@aspanner](https://github.com/aspanner)), Red Hat
```

Comment (TheFoxAtWork) [reply README.md:233]: small change to make so the URL works

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
small change

File: README.md (hunk 1 above)
Comment (aspanner) [reply README.md]: done.

**Review by anvega — APPROVED** [review]

**Pull request merged**